File: app/src/main/java/com/jeeps/gamecollector/model/ToggleCompletionResponse.java
package com.jeeps.gamecollector.model;

import java.io.Serializable;

public class ToggleCompletionResponse implements Serializable {
    private boolean completed;

    public ToggleCompletionResponse(boolean completed) {
        this.completed = completed;
    }

    public ToggleCompletionResponse() { }

    public boolean isCompleted() {
        return completed;
    }

    public void setCompleted(boolean completed) {
        this.completed = completed;
    }
}

File: app/src/main/java/com/jeeps/gamecollector/remaster/data/model/data/games/ToggleCompletionResponse.kt
package com.jeeps.gamecollector.remaster.data.model.data.games

import java.io.Serializable

data class ToggleCompletionResponse(
    @set:JvmName("setCompleted")
    @get:JvmName("isCompleted")
    var completed: Boolean = false
) : Serializable